src/app/scripts/cac/user/cac-user-preferences.js

```diff
@@ -76,6 +84,17 @@ CAC.User.Preferences = (function(Storages, _) {
     }
 
     /**
+     * Check if value has been set by user, or is a default.
+     * Will return false if given preference does not exist.
+     *
+     * @param {String} preference Name of setting to check
+     * @return {Boolean} True if getPreference will return a default value
+     */
+    function isDefault(preference) {
+        return !_.has(options, preference) && _.has(defaults, preference);
```

Comment:
Yes, that is what we want. We need to distinguish between whether it is the default value because it is returning a default, or if it happens to be the default because the user explicitly set it to that value. In the latter case, the options dialog needs to show the user selection and not the general menu text.

This user preferences refactor is to distinguish between the two.